android/fastlane/Fastfile

```diff
@@ -99,6 +112,19 @@ platform :android do
       notes_path: "fastlane/notes.txt",
       groups: options[:groups],
       notifications: options[:notify]
+      )
+  end
+
+  desc "Deploy to Play Store"
+  lane :deploy_play_store do |options|
+    upload_to_play_store(
+      track: 'beta',
+      apk: options[:apk],
+      json_key: options[:upload_key],
+      skip_upload_metadata: true,
```

Comment:
These skips can change when we have mechanisms in place, for now everything would be done on the Play Store website